src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToSubmitHandlerTest.java

```diff
@@ -35,110 +53,172 @@ public class AdjournCaseAboutToSubmitHandlerTest {
     @Mock
     private CaseDetails<SscsCaseData> caseDetails;
 
+    @Mock
+    private ListAssistHearingMessageHelper hearingMessageHelper;
+
+    @SuppressWarnings("unused")
+    @Mock
     private PreviewDocumentService previewDocumentService;
+
     private SscsCaseData sscsCaseData;
+    private AutoCloseable autoCloseable;
 
     @Before
     public void setUp() {
-        openMocks(this);
-        previewDocumentService = new PreviewDocumentService();
-        handler = new AdjournCaseAboutToSubmitHandler(previewDocumentService);
+        autoCloseable = openMocks(this);
+        ReflectionTestUtils.setField(handler, "isAdjournmentEnabled", true);
 
         when(callback.getEvent()).thenReturn(EventType.ADJOURN_CASE);
         when(callback.getCaseDetails()).thenReturn(caseDetails);
         sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId")
             .appeal(Appeal.builder().build())
+            .schedulingAndListingFields(SchedulingAndListingFields.builder()
+                .hearingRoute(HearingRoute.GAPS)
+                .build())
             .build();
         when(caseDetails.getCaseData()).thenReturn(sscsCaseData);
     }
 
+    @After
+    public void tearDown() throws Exception {
+        autoCloseable.close();
+    }
+
+    @DisplayName("Given a non adjourn case event, then return false")
     @Test
-    public void givenANonAdjournCaseEvent_thenReturnFalse() {
+    public void testNonAdjournCase() {
```

Comment:
Refer to PreviewDocumentServiceTest comments re: method naming. Apply to this class also.

Reply:
ok